internal/pkg/partition/constants.go

```diff
@@ -14,6 +14,7 @@ const (
 	EFISystemPartition  Type = "C12A7328-F81F-11D2-BA4B-00A0C93EC93B"
 	BIOSBootPartition   Type = "21686148-6449-6E6F-744E-656564454649"
 	LinuxFilesystemData Type = "0FC63DAF-8483-4772-8E79-3D69D8477DE4"
+	UserVolumePartition Type = "DAD6FD75-7A68-42AE-A92D-A9877FB083CF"
```

Comment:
is this a custom uuid?

Reply:
Yes, it's a random one, any suggestions on what we could reuse?

Reply:
I think we should use the same as linux filesystems type since in the end it's a linux partition , actually we can use the variable data partition uuid defined in UAPI https://uapi-group.org/specifications/specs/discoverable_partitions_specification/#partition-attribute-flags

Reply:
yep, thanks, makes sense, I will update

Reply:
I think I can go back to `LinuxFilesystemData`, as there is no direct match